appengine/go11x/cloudsql/cloudsql.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample cloudsql demonstrates connection to a Cloud SQL instance from App Engine standard.
+package main
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	// MySQL library, comment out for PostgreSQL
+	_ "github.com/go-sql-driver/mysql"
+	// PostgreSQL Library, uncomment to use.
+	// _ "github.com/lib/pq"
+)
+
+var db *sql.DB
+
+func main() {
+	db = GetDB()
+
+	http.HandleFunc("/", indexHandler)
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+		log.Printf("Defaulting to port %s", port)
+	}
+
+	log.Printf("Listening on port %s", port)
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+}
+
+// GetDB gets a connection to the database.
+func GetDB() *sql.DB {
```

Comment:
Drop the `Get` prefix? https://golang.org/doc/effective_go.html#Getters

Reply:
✅